staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -768,6 +772,20 @@ const (
 
 	// This annotation is to keep virt launcher container alive when an VMI encounters a failure for debugging purpose
 	KeepLauncherAfterFailureAnnotation string = "kubevirt.io/keep-launcher-alive-after-failure"
+
+	// MigrationTransportUnixAnnotation means that the VMI will be migrated using the unix URI
+	MigrationTransportUnixAnnotation string = "kubevirt.io/migrationTransportUnix"
```

Comment:
This doesn't seem to be part of the original PR, nor does it seem to be needed for the backported code.
Am I missing something?